src/sonic-yang-models/tests/yang_model_tests/tests/acl.json

```diff
@@ -0,0 +1,60 @@
+{
+    "ACL_RULE_UNDEFINED_PACKET_ACTION": {
+	"desc": "Configure undefined packet_action in ACL_RULE table.",
+	"eStr": ["Invalid value", "PACKET_ACTION"]
```

Comment:
If Libyang changes the Error string in the next version, We may have to change it at multiple places. I think, we should think a better way to write errors, so that we change only at few places later.

Reply:
I tried. But, couldn't find a way to do it from json files.

Reply:
I have added "eStrKey" which will be used to index into defaultYANGFailure to get list of strings to match. "eStr" would be appended to this to get the final list.